be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -25,10 +25,10 @@ object ElectionValidator extends MessageDataContentValidator with EventValidator
           Right(validationError(s"'start_time' (${data.start_time}) timestamp is smaller than 'created_at' (${data.created_at})"))
         } else if (!validateTimestampOrder(data.start_time, data.end_time)) {
           Right(validationError(s"'end_time' (${data.end_time}) timestamp is smaller than 'start_time' (${data.start_time})"))
-        } else if (expectedHash != data.id) {
+        } /*else if (expectedHash != data.id) {
```

Comment:
What is going on here ?

Reply:
I had a problem with the id calculation, so I changed some stuff around and visibly left the check there twice, once commented

Reply:
I still haven't been able to solve the id calculation error (which was coded before this semester), as the message examples we received didn't pass the validator even though they were generated by a frontend, so I left it commented for now